src/app/home/about/about.component.scss

```diff
@@ -0,0 +1,21 @@
+#layer {
+  background-color: rgba(100, 181, 246, 0.45);
+  height: 100%;
+  width: 100%;
+  position: absolute;
+  z-index: 1;
```

Comment:
Do not add a layer in this scenario. Currently we are not using any layer for parallax sections.